frontend/app/page.tsx

```diff
@@ -37,10 +38,21 @@ function Home() {
         >
           Play!
         </button>
+      </div>
+      <div>
         <button
           onClick={() => {
             router.push(`/rooms/create`);
```

Comment:
delete this button

Reply:
if we don't want to have the create private room page at `/rooms/create`

Reply:
will also need to update `frontend/app/rooms/random/page.tsx` to redirect to `/MakeRoom` instead of `"/rooms/create"` (I originally coded that page with that assumption)